src/main/resources/data/spectrum/recipes/mod_integration/create/enchantment_upgrade/capacity.json

```diff
@@ -0,0 +1,35 @@
+{
+  "type": "spectrum:enchantment_upgrade",
+  "group": "mod_compat",
```

Comment:
The field `group` is used to group and localize the unlock popups.
Since mod compat recipes are from various mods and can be about pretty much anything, they should not get grouped (else a potential bedrock knife and an enchanted book might be grouped in a single unlock popup)